specification/cognitiveservices/Language.Conversations/common.tsp

```diff
@@ -80,7 +88,7 @@ enum State {
 @added(Versions.v2023_04_01)
 model TaskIdentifier {
     @doc("task name")
-    taskName: string;
+    taskName?: string;
```

Comment:
How can `taskName` be optional when it's the only property?

Reply:
That is how we have it defined in the [json spec](https://github.com/Azure/azure-rest-api-specs/blob/4c38cb9966cd6afbb03c7e9b14997720a728baee/specification/cognitiveservices/data-plane/Language/stable/2023-04-01/common.json#L206)